HW_16/task_3.py

```diff
@@ -0,0 +1,65 @@
+class Product:
+    def __init__(self, type_, name, price):
+        self.type_ = type_
+        self.name = name
+        self.price = price
+
+    def __str__(self):
+        pass
+
+class ProductStore:
+    def __init__(self):
+        self.warehouse = {}
+        self.income = 0
+
+    def add(self, product: object, amount: int):
+        product.price = product.price * 1.3
+        self.warehouse[product.name] = {
+        "type_": product.type_,
```

Comment:
You don't need to put an underscore after "type" here - it is a valid string without `_`. You should put `_` after a variable name `type` so it wouldn't collide with the built-in `type`